test/tap/layouts-test.js

```diff
@@ -375,6 +375,119 @@ test('log4js layouts', (batch) => {
       assert.end();
     });
 
+    t.test('%m should apply util.format on data', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m',
+        "This isn't a test like other ones"
+      );
+      assert.end();
+    });
+
+    t.test('%m{1} should only consider data.slice(1)', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m{1}',
+        "isn't test"
+      );
+      assert.end();
+    });
+
+    t.test('%m{0,1} should behave like a dummy layout', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m{0,1}',
+        'This %s a %s like other ones'
+      );
+      assert.end();
+    });
+
+    t.test('%m{1,2} should only consider data.slice(1, 2)', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m{1,2}',
+        "isn't"
+      );
+      assert.end();
+    });
+
+    t.test('%m{1,2} should only consider data.slice(1, 2)', (assert) => {
+      const eventWithSeveralDataEntry = JSON.parse(JSON.stringify(event));
+      eventWithSeveralDataEntry.data = [
+        'This %s a %s like other ones',
+        "isn't",
+        'test',
+      ];
+      testPattern(
+        assert,
+        layout,
+        eventWithSeveralDataEntry,
+        tokens,
+        '%m{1,2}',
+        "isn't"
+      );
+      assert.end();
+    });
+
```

Comment:
Remove duplicated test.
```suggestion
```
There is an identical one from lines L432-L448.
https://github.com/log4js-node/log4js-node/blob/0632c2197ec193ca07d52a05946aae114a91e3ff/test/tap/layouts-test.js#L432-L448